manifests/project.pp

```diff
@@ -5,7 +5,21 @@
 # to to build up the commands list
 #
 define puppi::project (
-  $enable = true ) {
+  $deploy_root              = undef,
```

Comment:
All these parameters do not exist on https://github.com/example42/puppi/blob/master/manifests/project.pp